recipes-app/read-rtc-battery/files/src/readrtcbattery.c

```diff
@@ -0,0 +1,47 @@
+#include <sys/ioctl.h>
+#include <stdio.h>
+#include <linux/rtc.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+
+int main()
+{
+    int fd; // Device file path
+    int batteryStatus; // Status of battery 
+    int retval = 0; // Return value of ioctl function (-1 = Error)
+
+    // 1st step: Open Driver
+    printf("\nOpening Driver\n");
+    fd = open("/dev/rtc", O_RDONLY);
+
+    // 2nd step: Display error in command line if the device
+    // file wasn't found and close application
+    if(fd < 0) {

```

Comment:
`if` is not a function, therefore `if (...`

Reply:
corrected